rollup.config.js

```diff
@@ -0,0 +1,43 @@
+const commonjs = require('rollup-plugin-commonjs');
+const path = require('path');
+const resolve = require('rollup-plugin-node-resolve');
+const typescript = require('rollup-plugin-typescript');
+const pkg = require('./package.json');
+
+const selfId = path.resolve(__dirname, 'src/iife-self-placeholder.js');
+
+module.exports = {
+  input: 'src/index.ts',
+  external: [
+    selfId,
+    'whatwg-fetch',
+    'fake-xml-http-request',
+    'route-recognizer',
+  ],
+  output: [
+    {
+      name: 'Pretender',
+      file: pkg.main,
+      format: 'iife',
+      globals: {
+        [selfId]: 'self',
+        'whatwg-fetch': 'FakeFetch',
+        'fake-xml-http-request': 'FakeXMLHttpRequest',
+        'route-recognizer': 'RouteRecognizer',
+      },
+      banner: '//TODO: remove from source control when the tests begin using the source file instead of this one.\n' +
```

Comment:
I want to raise visibility on this code comment. The future direction I see Pretender taking is to run the tests by loading Pretender as a module (src/index.ts) instead of the iife.